docs/src/main/paradox/server/reflection.md

```diff
@@ -23,21 +23,33 @@ this metadata to implement things like completion and sending arbitrary
 commands.
 
 This is achieved by providing a gRPC service that provides endpoints that
-can be used to query this information
+can be used to query this information.
 
 ## Providing
 
-When providing gRPC Server Reflection, you will be serving multiple services
-(at least your own service and the reflection service) as described
-@ref[here](walkthrough.md) { #Serving_multiple_services }. The reflection service
-can be generated via `ServerReflection`:
+The generated service handler includes a convenience method to create an Akka HTTP 
+handler with your service together with Server Reflection: 
```

Comment:
```suggestion
handler for the service together with server reflection: 
```

Reply:
I think I like `handler for the service together with server reflection: ` better.

Reply:
:+1: LGTM, merge when you're happy!